man/man7/vdevprops.7

```diff
@@ -45,7 +45,7 @@ section, below.
 Every vdev has a set of properties that export statistics about the vdev
 as well as control various behaviors.
 Properties are not inherited from top-level vdevs, with the exception of
-checksum_n, checksum_t, io_n, io_t, slow_io_n, and slow_io_t.
+checksum_n, checksum_t, io_n, io_t, slow_io_downgrade, slow_io_n, and slow_io_t.
```

Comment:
Should here and below be `slow_io_reporting` instead of `slow_io_downgrade`?

Reply:
I agree